src/main/java/com/redhat/devtools/lsp4ij/client/features/LSPOnTypeFormattingFeature.java

```diff
@@ -35,8 +35,14 @@ public boolean isEnabled(@NotNull PsiFile file) {
             return false;
         }
 
+        // Make sure the feature is enabled in client config
+        LSPFormattingFeature formattingFeature = getClientFeatures().getFormattingFeature();
```

Comment:
We now consult that feature flag when deciding whether the feature itself is enabled.